src/Microsoft.ComponentDetection.Common/ComponentComparer.cs

```diff
@@ -1,5 +1,5 @@
 ﻿using System.Collections.Generic;
-using Microsoft.ComponentDetection.Contracts.TypedComponent;
+using Microsoft.ComponentDetection.Contracts.TypedComponentNS;
```

Comment:
Revert this namespace change and silence the linter warning if possible.

Reply:
done. removed all the duplicates in [this commit](https://github.com/microsoft/component-detection/pull/288/commits/fa8fce6651bb6d7fa8e5efa23707563b8b4ecb70)

Reply:
done. reverted all the namespace changes and suppressed the warning